Gemfile

```diff
@@ -43,6 +45,7 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```

Comment:
Maybe use gem 'pry', '~> 0.12.2' instead of byebug. And i think it will be better if we will use another one gem for development: gem 'annotate', '> 2.7'